godot_rl/wrappers/onnx/stable_baselines_export.py

```diff
@@ -120,15 +120,15 @@ def verify_onnx_export(ppo: PPO, onnx_model_path: str, num_tests=10, use_obs_arr
             obs = np.expand_dims(ppo.observation_space.sample(), axis=0)
             obs2 = torch.tensor(obs)
```

Comment:
```suggestion
            obs2 = torch.tensor(obs)
            obs = {"obs": obs}
```

This should return support for `use_obs_array=True` in verification.
That's used together with `SBGSingleObsEnv` and `MlpPolicy`. 
It should be OK to assume "obs" name as the default name as we did previously.

Feel free to reformat if there's some formatting error due to me pasting the line here.